apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx

```diff
@@ -98,18 +97,22 @@ export const UsernameValidationEditPage: FunctionComponent<UsernameValidationEdi
         }
 
         initializeForm();
-    }, [
-        validationData,
-        isValidationLoading
-    ]);
+    }, [ validationData, isValidationLoading ]);
 
     useEffect(() => {
-        if (initialFormValues !== undefined) {
+        if (initialFormValues) {
             setCurrentValues({ ...initialFormValues });
         }
 
     }, [ initialFormValues ]);
 
+    useEffect(() => {
+        if (currentValues) {
+            setTimeout(setPageLoaded, 200, true);
```

Comment:
IMO, setting an arbitrary timeout is not the correct way to solve this problem. AFAIU, the component is remounted during rendering the custom username validation config. We should find the RC for that remounting and fix it.

cc: @brionmario

Reply:
+1. Let's try to figure out why the issue is happening.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5217/commits/31442763afa9fa105ca1fc7c5878f2351708e9e0